RFCs/FS-1027-fsi-references.md

```diff
@@ -164,6 +162,18 @@ This will generate an error because `System.Whatever` version `4.1.0` will alrea
 
 We may consider using `AssemblyLoadContext` in some clever way in the future, but it's very difficult to get right and well outside the scope of what we need to support for a .NET Core 2.0 timeframe.
 
+## Handler resolution
+
+FSI/Design time support will look at the following places in order:
+
+* if the script is a physical file, check current folder and browse all parent folders looking for .fsharp/fsi-extensions folder in each one
+* look into ~/.fsharp/fsi-extensions
```

Comment:
I wonder if this should be called ``fsx-extensions``.  ``FSI.EXE`` is one tool implementing the F# scripting model (i.e. the ``.fsx`` script extension). These extensions are relevant to all such tools.